fitbenchmarking/fitbenchmarking
# 655 datafiles

#### Description of Work

Fixes #655 


#### Testing Instructions

1.  Check that running `fitbenchmarking` works.
2.  Check the updates to the docs make sense.
3.  Check the downloads linked to on the docs are there

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
